src/gigs-board/components/posts/Post.jsx

```diff
@@ -464,18 +465,20 @@ const postTitle =
   );
 
 const postExtra =
-  snapshot.post_type == "Sponsorship" ? (
+  snapshot.post_type == "Sponsorship" || parseInt(snapshot.amount) > 0 ? (
     <div key="post-extra">
       <h6 class="card-subtitle mb-2 text-muted">
-        Maximum amount: {snapshot.amount} {snapshot.sponsorship_token}
-      </h6>
-      <h6 class="card-subtitle mb-2 text-muted">
-        Supervisor:{" "}
-        <Widget
-          src={`neardevgov.near/widget/ProfileLine`}
-          props={{ accountId: snapshot.supervisor }}
-        />
+        Requested amount: {snapshot.amount} {snapshot.sponsorship_token}
       </h6>
+      {snapshot.supervisor && (
+        <h6 class="card-subtitle mb-2 text-muted">
+          Requested sponsor:{" "}
+          <Widget
+            src={`neardevgov.near/widget/ProfileLine`}
+            props={{ accountId: snapshot.supervisor }}
+          />
+        </h6>
+      )}
```

Comment:
I believe that changes in `Post` component can be completely reverted now since the data is provided through the description text now.